src/doc-components/GlobalVariable/globalVariable.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+import data from "../../data/globalVariables.yml"
+
+const GlobalVariable = props => {
```

Comment:
How does a doc component differ from other components

Reply:
they are components used exclusively in docs content